packages/timeline-state-resolver/package.json

```diff
@@ -104,7 +104,7 @@
     "sprintf-js": "^1.1.2",
     "superfly-timeline": "^8.2.1",
     "threadedclass": "0.8.3",
-    "timeline-state-resolver-types": "6.2.0-release37.4",
+    "@tv2media/timeline-state-resolver-types": "1.0.0-release37.5",
```

Comment:
I don't think it can be done this way, because now all the imports are conflicting with NRK.
It should look something like `"timeline-state-resolver-types": "npm:@tv2media/timeline-state-resolver-types@1.0.0-release37.5"`. It might make harder to link the packages during development, though, but we should rather find a workaround for that

Reply:
Updated